documentation/src/main/asciidoc/topics/ref_rest_security.adoc

```diff
@@ -1,7 +1,7 @@
 [id='rest_security']
 = REST Authentication
-Configure authentication to the REST endpoint with the `bin/user-tool.sh`
-script in the {brandname} server distribution.
+Configure authentication to the REST endpoint with the {brandname} command line
+interface (CLI) and the **user create** command.
```

Comment:
probably some link or reference to documentation/src/main/asciidoc/topics/proc_server_user_tool.adoc ?
Probably say the `user` is the command to use to create user/password that are able to access the REST endpoint...

Reply:
@pruivo thx!